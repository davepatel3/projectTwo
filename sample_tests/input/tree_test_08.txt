int
peek_nodes
peek_nodes
allocate
-84
0
remove
1
allocate
-79
0
remove
1
allocate
92
0
ref_node
1
allocate
57
0
ref_node
2
allocate
91
2
allocate
-41
2
allocate
-69
2
allocate
-98
2
set_parent
3
4
remove
3
set_parent
6
1
allocate
-7
0
peek_nodes
remove
5
remove
1
allocate
66
2
remove
2
allocate
32
0
allocate
-1
6
ref_node
6
set_parent
6
3
allocate
25
3
ref_node
3
remove
1
set_parent
4
6
remove
6
allocate
-1
0
ref_node
3
allocate
25
0
allocate
49
2
allocate
45
2
allocate
-57
5
ref_node
1
allocate
21
4
allocate
93
7
set_parent
6
8
remove
6
ref_node
2
allocate
67
3
allocate
-81
4
remove
5
set_parent
9
6
allocate
22
1
allocate
87
0
allocate
-52
1
allocate
1
0
set_parent
10
3
remove
9
ref_node
8
allocate
-46
0
allocate
67
8
allocate
-75
4
allocate
-57
10
ref_node
14
allocate
94
7
allocate
-98
14
allocate
25
7
peek_nodes
allocate
73
9
set_parent
18
5
remove
8
allocate
40
19
allocate
-36
6
allocate
76
17
remove
15
allocate
-19
2
set_parent
8
17
ref_node
2
allocate
23
10
allocate
95
10
remove
14
allocate
-62
7
allocate
38
19
allocate
98
6
allocate
-17
13
set_parent
4
18
allocate
-67
2
allocate
-59
4
set_parent
4
10
allocate
52
5
remove
3
allocate
-24
5
ref_node
17
allocate
-22
12
remove
19
set_parent
18
17
remove
17
allocate
87
0
allocate
-99
0
allocate
49
1
remove
5
ref_node
15
remove
16
set_parent
12
9
remove
15
allocate
38
22
remove
2
allocate
-9
21
remove
21
allocate
82
22
peek_nodes
remove
22
set_parent
6
0
allocate
-55
0
allocate
27
4
allocate
35
10
allocate
68
6
allocate
-95
12
allocate
-65
6
peek_nodes
ref_node
6
ref_node
3
ref_node
10
remove
3
allocate
22
4
set_parent
4
12
set_parent
4
9
allocate
9
0
remove
9
allocate
-53
8
allocate
-4
14
allocate
-34
14
ref_node
27
allocate
98
18
remove
6
remove
16
allocate
62
18
ref_node
15
allocate
-55
0
remove
18
allocate
-43
11
allocate
-35
0
ref_node
11
set_parent
1
23
remove
1
ref_node
11
ref_node
23
allocate
0
23
allocate
2
0
set_parent
11
2
ref_node
2
peek_nodes
allocate
50
0
set_parent
7
2
allocate
88
0
set_parent
2
20
ref_node
21
allocate
76
2
peek_nodes
set_parent
11
21
allocate
73
2
ref_node
20
set_parent
7
23
remove
24
ref_node
22
remove
7
ref_node
20
allocate
35
2
remove
23
allocate
-15
21
allocate
-72
0
ref_node
2
allocate
5
3
allocate
-11
2
set_parent
22
19
set_parent
10
2
allocate
-43
3
remove
20
remove
11
allocate
-68
21
remove
9
allocate
22
12
remove
21
ref_node
25
allocate
-24
0
allocate
-80
12
remove
14
peek_nodes
allocate
-73
0
ref_node
5
ref_node
25
allocate
-9
17
remove
25
set_parent
12
17
allocate
-45
5
ref_node
17
remove
8
allocate
-39
5
allocate
98
6
set_parent
10
94240241008243
ref_node
15
remove
12
ref_node
6
allocate
-59
15
set_parent
18
0
set_parent
17
6
remove
18
remove
5
allocate
12
0
allocate
-6
11
allocate
-92
1
allocate
-79
0
allocate
-87
0
quit
