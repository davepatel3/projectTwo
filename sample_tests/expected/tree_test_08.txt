peek_nodes
0: 0
peek_nodes
0: 0
allocate
1
remove
allocate
1
remove
allocate
1
ref_node
92
parent: 0
children: 
allocate
2
ref_node
57
parent: 0
children: 
allocate
3
allocate
4
allocate
5
allocate
6
set_parent
remove
set_parent
allocate
3
peek_nodes
0: 0
1: 92
2: 57
3: -7
4: -41
5: -69
6: -98
remove
remove
allocate
5
remove
allocate
6
allocate
1
ref_node
32
parent: 0
children: 1 
set_parent
allocate
4
ref_node
-7
parent: 0
children: 6 4 
remove
set_parent
remove
allocate
5
ref_node
-7
parent: 0
children: 
allocate
2
allocate
1
allocate
4
allocate
6
ref_node
49
parent: 2
children: 
allocate
7
allocate
8
set_parent
remove
ref_node
25
parent: 0
children: 1 4 
allocate
6
allocate
9
remove
set_parent
allocate
5
allocate
10
allocate
11
allocate
12
set_parent
remove
ref_node
93
parent: 7
children: 
allocate
9
allocate
13
allocate
14
allocate
15
ref_node
-75
parent: 4
children: 
allocate
16
allocate
17
allocate
18
peek_nodes
0: 0
1: 49
2: 25
3: -7
4: 45
5: 22
6: 67
7: 21
8: 93
9: -46
10: 87
11: -52
12: 1
13: 67
14: -75
15: -57
16: 94
17: -98
18: 25
allocate
19
set_parent
remove
allocate
13
allocate
8
allocate
20
remove
allocate
15
set_parent
ref_node
25
parent: 0
children: 1 4 15 
allocate
21
allocate
22
remove
allocate
20
allocate
8
allocate
17
allocate
14
set_parent
allocate
23
allocate
24
set_parent
allocate
25
remove
allocate
17
ref_node
-24
parent: 5
children: 
allocate
6
remove
set_parent
remove
allocate
21
allocate
22
allocate
16
remove
ref_node
-19
parent: 2
children: 
remove
set_parent
remove
allocate
20
remove
allocate
7
remove
allocate
24
peek_nodes
0: 0
1: [recycled]
2: [recycled]
3: [recycled]
4: [recycled]
5: [recycled]
6: -22
7: [recycled]
8: [recycled]
9: -46
10: [recycled]
11: [recycled]
12: 1
13: [recycled]
14: [recycled]
15: [recycled]
16: [recycled]
17: [recycled]
18: [recycled]
19: [recycled]
20: 38
21: [recycled]
22: -99
23: [recycled]
24: 82
25: [recycled]
remove
set_parent
allocate
4
allocate
10
allocate
3
allocate
14
allocate
13
allocate
8
peek_nodes
0: 0
1: [recycled]
2: [recycled]
3: 35
4: -55
5: [recycled]
6: -22
7: [recycled]
8: -65
9: -46
10: 27
11: [recycled]
12: 1
13: -95
14: 68
15: [recycled]
16: [recycled]
17: [recycled]
18: [recycled]
19: [recycled]
20: [recycled]
21: [recycled]
22: [recycled]
23: [recycled]
24: [recycled]
25: [recycled]
ref_node
-22
parent: 0
children: 14 8 
ref_node
35
parent: 10
children: 
ref_node
27
parent: 4
children: 3 
remove
allocate
19
set_parent
set_parent
allocate
18
remove
allocate
17
allocate
25
allocate
5
ref_node
Invalid handle!
allocate
16
remove
remove
allocate
15
ref_node
62
parent: 18
children: 
allocate
11
remove
allocate
1
allocate
23
ref_node
-55
parent: 0
children: 1 
set_parent
remove
ref_node
-55
parent: 0
children: 
ref_node
-35
parent: 0
children: 
allocate
2
allocate
7
set_parent
ref_node
0
parent: 23
children: 11 
peek_nodes
0: 0
1: [recycled]
2: 0
3: [recycled]
4: [recycled]
5: [recycled]
6: [recycled]
7: 2
8: [recycled]
9: [recycled]
10: [recycled]
11: -55
12: [recycled]
13: [recycled]
14: [recycled]
15: [recycled]
16: [recycled]
17: [recycled]
18: [recycled]
19: [recycled]
20: [recycled]
21: [recycled]
22: [recycled]
23: -35
24: [recycled]
25: [recycled]
allocate
21
set_parent
allocate
20
set_parent
ref_node
50
parent: 0
children: 
allocate
24
peek_nodes
0: 0
1: [recycled]
2: 0
3: [recycled]
4: [recycled]
5: [recycled]
6: [recycled]
7: 2
8: [recycled]
9: [recycled]
10: [recycled]
11: -55
12: [recycled]
13: [recycled]
14: [recycled]
15: [recycled]
16: [recycled]
17: [recycled]
18: [recycled]
19: [recycled]
20: 88
21: 50
22: [recycled]
23: -35
24: 76
25: [recycled]
set_parent
allocate
22
ref_node
88
parent: 0
children: 2 
set_parent
remove
ref_node
73
parent: 2
children: 
remove
ref_node
88
parent: 0
children: 2 
allocate
3
remove
allocate
13
allocate
12
ref_node
0
parent: 20
children: 22 3 
allocate
10
allocate
19
set_parent
set_parent
allocate
4
remove
remove
allocate
9
remove
allocate
25
remove
ref_node
22
parent: 12
children: 
allocate
5
allocate
14
remove
peek_nodes
0: 0
1: [recycled]
2: [recycled]
3: [recycled]
4: [recycled]
5: -24
6: [recycled]
7: [recycled]
8: [recycled]
9: [recycled]
10: [recycled]
11: [recycled]
12: -72
13: [recycled]
14: [recycled]
15: [recycled]
16: [recycled]
17: [recycled]
18: [recycled]
19: [recycled]
20: [recycled]
21: [recycled]
22: [recycled]
23: [recycled]
24: [recycled]
25: 22
allocate
17
ref_node
-24
parent: 0
children: 
ref_node
22
parent: 12
children: 
allocate
8
remove
set_parent
allocate
6
ref_node
-73
parent: 0
children: 8 12 
remove
allocate
16
allocate
15
set_parent
Invalid handle!
ref_node
98
parent: 6
children: 
remove
ref_node
-45
parent: 5
children: 15 
allocate
18
set_parent
set_parent
remove
remove
allocate
1
allocate
Node is recycled!
allocate
24
allocate
7
allocate
23
quit
